index.html

```diff
@@ -802,9 +849,34 @@
     {name: "java"},
     {name: "cpp"},
     {name: "python"},
-    {name: "go", tabSize: 4, indentWithTabs: true, indentUnit: 4}
+    {name: "go", tabSize: 2, indentWithTabs: true, indentUnit: 4}
```

Comment:
Are you sure you want to set different sizes for tab and the indentation unit? It'll use two tabs for every indentation level like this, which is probably not a good idea.

Reply:
my mistake. fixed.